roles/nginx/vars/Debian.yml

```diff
@@ -1,3 +1,4 @@
 ---
 nginx_user: www-data
 nginx_default_vhost_path: /etc/nginx/sites-enabled/default
+nginx_default_vhost_docroot: "/var/www/html"
```

Comment:
there's already a defaulted var (badly) named `root` that exist in the role

Reply:
that looks like not even used, let me try to cleanup